### ISSUE TITLE
Larger batch size

### ISSUE THREAD

**Issue by mshubhankar** [awslabs/fast-differential-privacy#41]
Is there a recommended way to train with larger batch sizes using FastDP and a smaller physical memory GPU? 
I am searching for a technique similar to the [BatchMemoryManager](https://opacus.ai/api/batch_memory_manager.html) from Opacus or standard non-private techniques such as [here](https://kozodoi.me/blog/20210219/gradient-accumulation).

**Comment by woodyx218** [comment]
You can use gradient accumulation just like non-DP training, e.g. Line 96-99 in `CIFAR_TIMM.py`. This has nothing to do with DP engine so I am closing this issue. Feel free to reopen if you have further questions.
